From 0da475bb8f81ce162472eb40efb31bc41d2bef60 Mon Sep 17 00:00:00 2001
From: Vincent Tinelli <vincent.tinelli@intel.com>
Date: Fri, 2 May 2014 21:10:37 +0200
Subject: [PATCH 310/429] mmc: allow up to 32 block devices

kernel only enumerates 9 partitions as CONFIG_MMC_BLOCK_MINORS=10
set it to 32 to allow using more partitions

Signed-off-by: Vincent Tinelli <vincent.tinelli@intel.com>
---
 arch/x86/configs/i386_edison_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/configs/i386_edison_defconfig b/arch/x86/configs/i386_edison_defconfig
index 807af6eca0f5..2b943c96ba43 100644
--- a/arch/x86/configs/i386_edison_defconfig
+++ b/arch/x86/configs/i386_edison_defconfig
@@ -2669,7 +2669,7 @@ CONFIG_MMC_UNSAFE_RESUME=y
 # MMC/SD/SDIO Card Drivers
 #
 CONFIG_MMC_BLOCK=y
-CONFIG_MMC_BLOCK_MINORS=10
+CONFIG_MMC_BLOCK_MINORS=32
 CONFIG_MMC_BLOCK_BOUNCE=y
 # CONFIG_SDIO_UART is not set
 # CONFIG_MMC_TEST is not set
-- 
2.37.3

